### ISSUE TITLE
Question about adj_full_norm

### ISSUE THREAD

**Issue by JunweiSUN** [GraphSAINT/GraphSAINT#32]
Dear authors, thanks for publishing this great repo!

I have a small question:

In the pytorch version [code](https://github.com/GraphSAINT/GraphSAINT/blob/master/graphsaint/pytorch_version/train.py#L52)，the full adj is normalized and used to generate the minibatch. However, in an inductive setting, one may only see the training data. Is it more appropriate to normalize the train/val/test nodes separately, or I misunderstand the code here? Looking forward to your reply! Thank you very much!

**Comment by tedzhouhk** [comment]
Hi Junwei,

In our minibatch, we only use the row and column indices of the adj_train matrix to generate minibatch for training.  The edge weights in the sampled subgraph are filled by the normalized probability of an edge being sampled in the subgraph (See Proposition 3.1 in the ICLR paper and self.norm_aggr_train in Minibatch class). The adj_full_norm passed to the Minibatch class is only used to generate the validation and test batch.
Best,
Hongkuan

**Comment by JunweiSUN** [comment]
@tedzhouhk thanks a lot！
